src/conn.h

```diff
@@ -35,6 +35,7 @@ struct conn
 	struct message response;                /* Response message meta data */
 	struct handle handle;
 	bool closed;
+	bool await_message; /* Conn is waiting for a message */
```

Comment:
Maybe `reading_message` or something instead?

Reply:
It's not really reading the message, it's waiting for a message.

Reply:
hmm, okay in the end I agree, `reading_message` is more clear :-)